app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -513,15 +513,11 @@ function AcuantCapture(
 
   function onSelfieCaptureOpen() {
     trackEvent('idv_sdk_selfie_image_capture_opened', { captureAttempts });
-
-    setImageCaptureText('');
     setIsCapturingEnvironment(true);
   }
 
   function onSelfieCaptureClosed() {
     trackEvent('idv_sdk_selfie_image_capture_closed_without_photo', { captureAttempts });
-
-    setImageCaptureText('');
```

Comment:
I'm not actually removing this, it's just moving into the acuant-selfie-camera `onClosed` function